tap_teamwork/streams/tickets.py

```diff
@@ -1,14 +1,25 @@
-from typing import Dict, Iterator, List
+from typing import Dict, Iterator, List, Optional
 from singer import get_logger
-from tap_teamwork.streams.abstracts import FullTableStream
+from tap_teamwork.streams.abstracts import IncrementalStream
 
 LOGGER = get_logger()
 
-
-class Tickets(FullTableStream):
+class Tickets(IncrementalStream):
     tap_stream_id = "tickets"
     key_properties = ["id"]
-    replication_method = "FULL_TABLE"
-    replication_keys: List[str] = []
+    replication_method = "INCREMENTAL"
+    replication_keys: List[str] = ["updatedAt"]
     data_key = "tickets"
     path = "desk/v2/tickets.json"
+    children = ["ticket_details"]
```

Comment:
I think this will make thing easier that list of strings.

```suggestion
    children = [TicketDetails]
```